override.js

```diff
@@ -1,7 +1,7 @@
 /* -*- Mode: Java; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
 /* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
 
-'use strict';
+// 'use strict';
```

Comment:
> Why is this commented out?

Seems unnecessary, in my testing; commented back in, in https://github.com/mykmelez/j2me.js/commit/f367a71386b5b4fc31f7142a8a265c8be2ba53dc.